src/components/Disputes/DisputeInfo.js

```diff
@@ -75,15 +75,6 @@ const DisputeInfo = React.memo(function({
           if (loading) {
```

Comment:
We can probably move `loading` down to be after `isDisputeVoided` as well

Reply:
In this case i don't think there's a difference since we won't know if the dispute is voided or not until `loading` ends. The loading is not for the `ipfs content` but for the `graph subscription`.

Reply:
Gotcha, this is specifically in the case where you open a URL with a dispute id and we're waiting to fetch information?

But in either case, we know the dispute id immediately, so it seems like we should be able to immediately prefer showing the voided state?